application/controllers/ModelController.php

```diff
@@ -511,6 +511,10 @@ public function createAction()
                 );
                 $this->view->errorFlag = true;
                 return;
+            } else if (!Erfurt_Uri::check($newModelUri)) {
+                $this->_owApp->appendMessage(
+                    new OntoWiki_Message('The given String is no legal URI (http://...)', OntoWiki_Message::ERROR)
```

Comment:
Since we are aiming for IRI support, it is better to write IRI or URI (resp. other way round)